core/src/main/scala/kafka/server/KafkaRaftController.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package kafka.server
+
+class KafkaRaftController {
```

Comment:
Hmm, I found it a little strange if "Raft" is the only prefix. The `RaftController` is not controller Raft... I've considered calling it `MetadataController` previously. To go along with that, we could have `DataBroker` or just `Broker`.

Reply:
Yeah, I agree that it reads a bit strange without the `Kafka` prefix. I think what you have in the PR is fine.

Reply:
How about RaftBrokerServer and ControllerServer?  That would help differentiate this from the Controller / QuorumController classes.  It would also be consistent with KafkaServer.

I think it's also more accurate-- these classes manage "all the server stuff" including socket server, etc.

Reply:
Or maybe just `Controller` and `Broker`?

Reply:
Decided to go with `ControllerServer` and `BrokerServer`. As Colin mentioned offline, the "server" suffix is helpful conveying that this component is responsible for starting sockets and such. This distinguishes `ControllerServer` from the actual implementation of the controller logic.